spec/factories/patients.rb

```diff
@@ -0,0 +1,14 @@
+FactoryBot.define do
+  factory :patient do
+    common_names = { female: %w[anjali divya ishita priya priyanka riya shreya tanvi tanya vani],
```

Comment:
Let's try and abstract these somewhere for re-use. I can see us wanting names in the `User` model as well.